lib/build_pack.rb

```diff
@@ -0,0 +1,12 @@
+require 'fileutils'
+require 'net/http'
+
+module BuildPack
+  def self.run(build_dir, cache_dir)
+    Installer.install(build_dir, cache_dir)
+  end
+end
+
+require_relative "build_pack/installer"
+require_relative "build_pack/downloader"
+require_relative "build_pack/logger"
```

Comment:
I was following what [heroku-buildpack-ruby](https://github.com/heroku/heroku-buildpack-ruby/blob/master/lib/language_pack.rb#L29-L46) was doing, but, again, perhaps their use case was more complex, so I've moved them to the top (and I agree, apart from the  ruby buildpack I don't think I've seen requires anywhere but at the top).  Also, big 👍  to scoping the requires as narrowly as possible, I forgot about these when I refactored.